### PR TITLE
Fix new format links for MejorTorrent

### PR DESCRIPTION
**Pull request by alvaro654**

### CODE REVIEW

File: src/Jackett.Common/Indexers/MejorTorrent.cs [hunk 1]
```diff
@@ -156,7 +156,7 @@ interface IScraper<T>
 
         class RssScraper : IScraper<IEnumerable<KeyValuePair<MTReleaseInfo, Uri>>>
         {
-            private readonly string LinkQuerySelector = "a[href*=\"/serie\"]";
+            private readonly string LinkQuerySelector = "a[href*=series_extend]";
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/MejorTorrent.cs]: can you provide a link to a page that has this href, I don't seem to be able to find it.

Comment (alvaro654) [reply src/Jackett.Common/Indexers/MejorTorrent.cs]: Yes, of course. 

On http://www.mejortorrentt.org/secciones.php?sec=ultimos_torrents which is used for last releases, on TV series releases links changed from /serie...... to:

"<a href="secciones.php?sec=descargas&amp;ap=series_extend&amp;id1=59534&amp;id2=59535&amp;titulo=NCIS-New-Orleans-5-Temporada" style="font-size:11px;">NCIS: New Orleans - 5ª Temporada: 5x23 -</a>"

In order to allow the indexer find new releases is needed to modify the query.
There is also a problem with the indexer searching tv series by season and episode that I expect to fix as soon as posible. Working on

**Comment by garfield69** [comment]
thanks.

**Comment by DAVIZINH0** [comment]
hello! if i tried to add this indexer i received this error:

One or more errors occurred. (Imposible to detect episode ID in RSS) (Imposible to detect episode ID in RSS) (Imposible to detect episode ID in RSS) (Imposible to detect episode ID in RSS) (Imposible to detect episode ID in RSS): Imposible to detect episode ID in RSS

**Comment by garfield69** [comment]
@DAVIZINH0 version of jackett?

**Comment by DAVIZINH0** [comment]
Jackett Version 0.11.608.0 
in docker with container linuxserver/jackett::development

**Comment by garfield69** [comment]
oh well. I get sequence contains no element.
so I guess it is still broken.

**Comment by alvaro654** [comment]
Check you are using www.mejortorrentt.org, besides of patching I have found that if you have been trying to add this indexer before the fix the website could have banned your cookie, so it is needed to start a new container or delete your cookie id data. I have just started a new container linux-server/jackett-development and it is working for me

**Comment by DAVIZINH0** [comment]
ok, i understand. how can i delete my cookie id data? its easy?

**Comment by alvaro654** [comment]
First try working with a new container. However I recomend you waiting for next update because I still having a problem geting download links because now the page provides zip compressed torrent files.

**Comment by SferaDev** [comment]
Even though the indexer works fine on Jackett, Sonarr fails to get data from it during setup.

Message
```
Unable to connect to indexer: Torznab error detected: System.ArgumentNullException: Value cannot be null.
Parameter name: value
   at System.Xml.Linq.XAttribute..ctor(XName name, Object value)
   at Jackett.Common.Models.ResultPage.<ToXml>b__13_0(ReleaseInfo r) in /home/appveyor/projects/jackett/src/Jackett.Common/Models/ResultPage.cs:line 63
   at System.Linq.Enumerable.SelectListIterator`2.MoveNext()
   at System.Xml.Linq.XContainer.AddContentSkipNotify(Object content)
   at System.Xml.Linq.XContainer.AddContentSkipNotify(Object content)
   at Jackett.Common.Models.ResultPage.ToXml(Uri selfAtom) in /home/appveyor/projects/jackett/src/Jackett.Common/Models/ResultPage.cs:line 39
   at Jackett.Server.Controllers.ResultsController.Torznab(TorznabRequest request) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 417
```

Exception

```
NzbDrone.Core.Indexers.Torznab.TorznabException: Torznab error detected: System.ArgumentNullException: Value cannot be null.
Parameter name: value
   at System.Xml.Linq.XAttribute..ctor(XName name, Object value)
   at Jackett.Common.Models.ResultPage.<ToXml>b__13_0(ReleaseInfo r) in /home/appveyor/projects/jackett/src/Jackett.Common/Models/ResultPage.cs:line 63
   at System.Linq.Enumerable.SelectListIterator`2.MoveNext()
   at System.Xml.Linq.XContainer.AddContentSkipNotify(Object content)
   at System.Xml.Linq.XContainer.AddContentSkipNotify(Object content)
   at Jackett.Common.Models.ResultPage.ToXml(Uri selfAtom) in /home/appveyor/projects/jackett/src/Jackett.Common/Models/ResultPage.cs:line 39
   at Jackett.Server.Controllers.ResultsController.Torznab(TorznabRequest request) in /home/appveyor/projects/jackett/src/Jackett.Server/Controllers/ResultsController.cs:line 417
  at NzbDrone.Core.Indexers.Torznab.TorznabRssParser.PreProcess (NzbDrone.Core.Indexers.IndexerResponse indexerResponse) [0x000b6] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\Torznab\TorznabRssParser.cs:42 
  at NzbDrone.Core.Indexers.RssParser.ParseResponse (NzbDrone.Core.Indexers.IndexerResponse indexerResponse) [0x0000d] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\RssParser.cs:55 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].FetchPage (NzbDrone.Core.Indexers.IndexerRequest request, NzbDrone.Core.Indexers.IParseIndexerResponse parser) [0x00050] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:308 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x0000e] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:335 
```

**Comment by alvaro654** [comment]
That is because currently the links use a http post method to get a compressed torrent file, there is no link to file so generates null link. Trying to find a solution.

**Comment by rovik0** [comment]
Jackett 0.12.1014.0 works just fine but with Sonarr 2.0.0.5338 fails and at the log file returns the same as  @SferaDev

**Pull request merged**